src/main/java/com/conversions/BinaryToGray.java

```diff
@@ -0,0 +1,30 @@
+package src.main.java.com.conversions;
+
+public class BinaryToGray
+{
+	/*	This method convert the binary number into gray code
+		@param binaryno need to convert binary number into gray code
+		@return graycode return as string
+	*/
+
+	public String binaryToGray(String binarycode)
+	{
+				
+		String graycode=Character.toString(binarycode.charAt(0));
```

Comment:
Add single space on both sides of the `=`

```suggestion
		String graycode = Character.toString(binarycode.charAt(0));
```